charts/airbyte/values.yaml

```diff
@@ -721,7 +721,7 @@ temporal:
   image:
     repository: temporalio/auto-setup
     pullPolicy: IfNotPresent
-    tag: "1.7.0"
+    tag: "latest"
```

Comment:
i'm not sure we can do this. i remember us running into issues here. [See this](https://github.com/airbytehq/airbyte/tree/master/airbyte-temporal#testing-a-temporal-migration) for more info.

This is also making me realise this isn't correct. Our docker files use a [wrapped Temporal image](https://github.com/airbytehq/airbyte/blob/master/airbyte-temporal/Dockerfile#L2) and we should probably use that in Helm too, and hopefully upgrade it.

Reply:
Bumped version the same as specified in our dockerfiles